topostats/grains.py

```diff
@@ -41,7 +41,7 @@ def __init__(
             "lower": [None, None],
         },
         direction: str = None,
-        absolute_smallest_grain_size: float = None,
+        smallest_grain_size: float = None,
```

Comment:
If changing name to reflect units...

```suggestion
        smallest_grain_size_nm: float = None,
```

Docstring should be updated too but can't comment on unchanged lines.